sources/packages/backend/apps/queue-consumers/src/processors/assessment/cancel-application-assessment.processor.ts

```diff
@@ -58,6 +58,20 @@ export class CancelApplicationAssessmentProcessor {
       throw new Error(errorMessage);
     }
 
+    if (
+      assessment.studentAssessmentStatus !==
+      StudentAssessmentStatus.CancellationQueued
+    ) {
+      await job.discard();
+      const warnMessage = `Assessment id ${job.data.assessmentId} has already been cancelled.`;
```

Comment:
I know, Ideally, only `CancellationQueued` assessment will come at this stage, but we are checking `assessment.studentAssessmentStatus !==
      StudentAssessmentStatus.CancellationQueued`,
 IMO, `Assessment id ${job.data.assessmentId} is not in expected status.`, will be better

Reply:
I change the messages to 
`Assessment id ${job.data.assessmentId} is not in ${StudentAssessmentStatus.CancellationQueued} status.` and 
`Workflow process not executed due to the assessment cancellation not being in the correct status.`